README.rst

```diff
@@ -58,6 +58,10 @@ Usage
 
     tutor [dev|local] do load-xapi-test-data
 
+#. (Optional) Sink course data from mongoDB to clickhouse (see  `https://github.com/openedx/openedx-event-sink-clickhouse` for more information)::
```

Comment:
nit: "(Optional) Import course data from the LMS to ClickHouse"

Technically it uses a combination of LMS data sources and caches. 🤷